src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -43,10 +43,17 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         &mut self,
         instance: ty::Instance<'tcx>,
         args: &[OpTy<'tcx, M::PointerTag>],
-        dest: PlaceTy<'tcx, M::PointerTag>,
+        dest: Option<PlaceTy<'tcx, M::PointerTag>>,
```

Comment:
why was this change done?

Reply:
This allows us to handle the `abort` intrinsic, which doesn't have a dest, in Miri.

Reply:
This is not pretty, I wonder if we could use a "null place" or so instead.

But we can come back to that later.